### DISCUSSION TITLE
Tag manifest is queued to be scanned

### DISCUSSION THREAD

**Discussion by sbenhai** [quay/clair#1304] [Q&A]
I'm trying to scan some image using Clair on my GCP machine, but every image I used got the same issue:
"_The manifest for this tag is queued to be scanned for vulnerabilities_" ([link](https://drive.google.com/file/d/1QjDMTXiydh8KiUvU2tdBxcR2faZfXwFh/view?usp=sharing))

When I'm trying to use ClairCTL I get "502 Bad Gateway":

`
instance-2:~$ ./clairctl-linux-amd64 -D report ubuntu
2021-06-28T14:53:20Z DBG fetching ref=ubuntu
2021-06-28T14:53:20Z DBG using text output
2021-06-28T14:53:21Z DBG found manifest digest=sha256:376209074d481dca0a9cf4282710cd30a9e7ff402dea8261acdaaf57a18971dd ref=ubuntu
2021-06-28T14:53:21Z DBG  method=GET path=/indexer/api/v1/index_report/sha256:376209074d481dca0a9cf4282710cd30a9e7ff402dea8261acdaaf57a18971dd status="502 Bad Gateway"
2021-06-28T14:53:21Z DBG index error error="unexpected return status: 502" ref=ubuntu
2021-06-28T14:53:21Z ERR  error="unexpected return status: 502"
`

**Comment by crozzy** [comment]
> I'm trying to scan some image using Clair on my GCP machine, but every image I used got the same issue:
> "_The manifest for this tag is queued to be scanned for vulnerabilities_" ([link](https://drive.google.com/file/d/1QjDMTXiydh8KiUvU2tdBxcR2faZfXwFh/view?usp=sharing))

It's a little hard to debug this without the logs from the quay container pertaining to it's communication with clair. You should check them to see if quay gives some more info.
> 
> When I'm trying to use ClairCTL I get "502 Bad Gateway":
> 
> `instance-2:~$ ./clairctl-linux-amd64 -D report ubuntu 2021-06-28T14:53:20Z DBG fetching ref=ubuntu 2021-06-28T14:53:20Z DBG using text output 2021-06-28T14:53:21Z DBG found manifest digest=sha256:376209074d481dca0a9cf4282710cd30a9e7ff402dea8261acdaaf57a18971dd ref=ubuntu 2021-06-28T14:53:21Z DBG method=GET path=/indexer/api/v1/index_report/sha256:376209074d481dca0a9cf4282710cd30a9e7ff402dea8261acdaaf57a18971dd status="502 Bad Gateway" 2021-06-28T14:53:21Z DBG index error error="unexpected return status: 502" ref=ubuntu 2021-06-28T14:53:21Z ERR error="unexpected return status: 502"`

For the `report` command `clairctl` is acting as a client to a running clair instance, hence you will need to specify a clair host (if it differs from the default `http://localhost:6060/`).

**Comment by hdonnay** [comment]
Also note that `ubuntu` is taken to mean `docker.io/library/ubuntu:latest`, which says very little about what's happening with quay-clair communication.

**Comment by sbenhai** [comment]
When I'm trying to go Step by step with the tutorial, I get this [error](https://drive.google.com/file/d/1NjS9wEnsSU_F9EwxlOi2IF-suPWygxRv/view?usp=sharing) at Step 4
